automated install/basic-install.sh

```diff
@@ -300,10 +300,19 @@ package_manager_detect() {
         # Check for and determine version number (major and minor) of current php install
         local phpVer="php"
         if is_command php ; then
-            printf "  %b Existing PHP installation detected : PHP version %s\\n" "${INFO}" "$(php <<< "<?php echo PHP_VERSION ?>")"
-            printf -v phpInsMajor "%d" "$(php <<< "<?php echo PHP_MAJOR_VERSION ?>")"
-            printf -v phpInsMinor "%d" "$(php <<< "<?php echo PHP_MINOR_VERSION ?>")"
-            phpVer="php$phpInsMajor.$phpInsMinor"
+            phpVer="$(php -v 2> /dev/null | head -n1 | cut -d '-' -f1 | cut -d ' ' -f2)"
+            # Check if the first character of the string is numeric
```

Comment:
I don't know what a corrupt `php` setup will return for `php -v`. I do know that the call to `php` echoing `PHP_VERSION` will give us output that we already know about. Plus that call doesn't need any pipes or cuts.